department-of-veterans-affairs/va.gov-cms
# VACMS-12787 Move facility events to the appropriate facility modules.

## Description

closes #12787

- This moves event related logic that applies to Multiple Types of Facilities out of va_gov_backend and into va_gov_facilities.  
- This moves event related logic that applies to VAMC Facilities out of va_gov_backend and into va_gov_vamc.  

## Testing done


## Screenshots


## QA steps

- login as RyanSt (pittsburgh)


Code moved: Show "undefined" when EHR number is empty.

- Navigate to https://pr13066-j88djnxetcs6zem67urwoapbawmab1rj.ci.cms.va.gov/node/318/edit
- [x] Validate that  "Undefined" shows as the "VA Health Connect phone number
![image](https://user-images.githubusercontent.com/5752113/227969136-3eeea357-8c53-4988-85eb-f06622087fc7.png)


Code moved: Build health service description array in settings
- Navigate to https://pr13066-j88djnxetcs6zem67urwoapbawmab1rj.ci.cms.va.gov/node/38986/edit
- open your browser console
- enter `drupalSettings.availableHealthServices`
- Validate that you see an array of arrays of services.
![image](https://user-images.githubusercontent.com/5752113/227973491-d6de1420-1792-4b5e-8a46-df86f7e99a91.png)


Code moved: Build health service form intro
- Navigate to https://pr13066-j88djnxetcs6zem67urwoapbawmab1rj.ci.cms.va.gov/node/38986/edit
- [x] Validate that this text appears beneath the heading
![image](https://user-images.githubusercontent.com/5752113/227971162-1efab2f0-9b3b-4d8e-9345-88bc71167a8c.png)


Code moved: Remove delete button from facilities
- Navigate to https://pr13066-j88djnxetcs6zem67urwoapbawmab1rj.ci.cms.va.gov/node/87/edit
- [x] Validate there is no "Delete" tab at the top of the page.
- [x] Validate there is no "Delete" button at the bottom of the form (near the save button) 



### Definition of Done

### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
